File: lua/refactoring/tests/refactor/106/go/struct-return/extract.start.go
package main

import "fmt"

type foobar struct {
   poggers int
}

func (f *foobar) simple_function(a int) (int, int) {
    var test int = 1
    test_other := 1
    for idx := test - 1; idx < test_other; idx++ {
        fmt.Println(f.poggers, idx, a)
    }

    return test, test_other
}

File: lua/refactoring/tests/refactor/106/php/class/extract.start.php
<?php

class RefuctorPlease
{
    public function multiple_returns($a)
    {
        $test = 1;
        $test_other = 11;
        for ($idx = $test - 1; $idx < $test_other; ++$idx) {
            print $idx;
            print $a;
        }

        return $test_other;
    }
}
